### PR TITLE
Add more logs

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Review by fultonj — COMMENTED** [review]
+2

File: controllers/openstackdataplanenode_controller.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ type OpenStackDataPlaneNodeReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/reconcile
 func (r *OpenStackDataPlaneNodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
 	r.Log = log.FromContext(ctx)
-	r.Log.Info("Reconciling")
+	r.Log.Info("Reconciling Node(s)")
```
Comment (slagle) [inline controllers/openstackdataplanenode_controller.go:80]: why the plural? this only reconciles a single node

File: controllers/openstackdataplanerole_controller.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ type OpenStackDataPlaneRoleReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/reconcile
 func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
 	r.Log = log.FromContext(ctx)
-	r.Log.Info("Reconciling")
+	r.Log.Info("Reconciling Role(s)")
```
Comment (slagle) [inline controllers/openstackdataplanerole_controller.go:71]: ditto

Comment (fao89) [reply controllers/openstackdataplanerole_controller.go]: done!

**Review by slagle — APPROVED** [review]

**Pull request merged**